mountpoint-s3/tests/fuse_tests/mod.rs

```diff
@@ -222,3 +225,17 @@ mod s3_session {
         }
     }
 }
+
+/// Take a `read_dir` iterator and return the entry names
+pub fn read_dir_to_entry_names(read_dir_iter: ReadDir) -> Vec<String> {
+    read_dir_iter
+        .map(|entry_result| entry_result.expect("no io err during readdir"))
```

Comment:
nit: is there a reason to have multiple `map`s here?

Reply:
It doesn't matter either way - only to break into logical steps which we can do with local variables as well.

Happy to change if you would find that more readable.

Reply:
I'll push it anyway as a single map